model/relabel/relabel.go

```diff
@@ -183,17 +188,22 @@ func (re Regexp) MarshalYAML() (interface{}, error) {
 // are applied in order of input.
 // If a label set is dropped, nil is returned.
 // May return the input labelSet modified.
-func Process(labels labels.Labels, cfgs ...*Config) labels.Labels {
+//
+// Partially applied relabels will be returned if an error occurs.
+func Process(labels labels.Labels, cfgs ...*Config) (labels.Labels, error) {
 	for _, cfg := range cfgs {
-		labels = relabel(labels, cfg)
-		if labels == nil {
-			return nil
+		result, err := relabel(labels, cfg)
+		if result == nil {
+			return nil, nil
```

Comment:
```suggestion
			return nil, nil
```

err could still be != nil here